webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -369,8 +401,19 @@ func (v validator) validateImageOnCreate(ctx *pkgctx.WebhookRequestContext, vm *
 	case vmopv1util.IsImagelessVM(*vm) &&
 		(pkgcfg.FromContext(ctx).Features.VMImportNewNet ||
 			pkgcfg.FromContext(ctx).Features.VMIncrementalRestore):
+		// TODO: Simplify this once mobility operator starts creating VMs with correct annotation.
```

Comment:
I am fine just removing the FSS checks.  However, you need to add a check to ensure only privileged users can set the annotations for Register/Import.